docs/advanced_usage.md

```diff
@@ -90,11 +90,13 @@ subsection.
 In case of potential compromise, notary provides a CLI command for rotating keys. Currently, you can use the `notary key rotate` command to rotate the targets or snapshot keys.
 
 While the snapshot key is managed by the notary client by default, use the `notary key
```

Comment:
It looks like your documentation appears in the next paragraph.  :)  This just needed a rebase.